contracts/vaults/Withdrawals.sol

```diff
@@ -68,7 +78,17 @@ contract Withdrawals is Base {
         return shares;
     }
 
-    /** @dev See {IERC4626-redeem}. */
+    /** 
+    * @notice Redeem shares in the vault, and receive the respective amount of
+    * underlying assets.
+    * Can only be performed if a withdraw request has been previously
+    * submitted, and the pending period had passed, and while the withdraw
+    * enabled timeout had not passed.
```

Comment:
https://github.com/hats-finance/hats-contracts/pull/286/files#r950941186

Reply:
Also only when not safety period and no active claim exists